geospaas_harvesting/ingesters.py

```diff
@@ -508,29 +509,30 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         normalized_attributes = {}
         n_points = int(kwargs.get('n_points', 10))
         nansat_options = kwargs.get('nansat_options', {})
+        url_scheme = urlparse(dataset_info).scheme
+        if not 'http' in url_scheme and not 'ftp' in url_scheme:
+            normalized_attributes['geospaas_service_name'] = FILE_SERVICE_NAME
+            normalized_attributes['geospaas_service'] = LOCAL_FILE_SERVICE
+        elif 'http' in url_scheme and not 'ftp' in url_scheme:
+            normalized_attributes['geospaas_service_name'] = DAP_SERVICE_NAME
+            normalized_attributes['geospaas_service'] = OPENDAP_SERVICE
+        elif 'ftp' in url_scheme:
+            raise ValueError("LOCALHarvester (which uses NansatIngester) is only for local file"
+        " addresses or http addresses, not for ftp protocol")
```

Comment:
I don't think this is necessary, Nansat will probably raise it's own exception if you try to open an FTP URL with it.

Reply:
It does not harm anything! let it be here to capture any wrong input and prevent processing it.